preproc/ft_preproc_dftfilter.m

```diff
@@ -1,27 +1,55 @@
-function [filt] = ft_preproc_dftfilter(dat, Fs, Fl)
+function [filt] = ft_preproc_dftfilter(dat, Fs, Fl, Flreplace, Flwidth, Neighwidth)
```

Comment:
for  backward compatibility: could you make the new additional input arguments optional, e.g. 

ft_preproc_dftfilter(dat, Fs, Fl, varargin)

if numel(varargin)>0
  Flreplace = varargin{1};
  Flwidth etc
else 
  Flreplace = 'zero';
end